app/views/blog_posts/index.rb

```diff
@@ -29,10 +29,27 @@ def blog_items
       mustache[:blog_items] ||= @blog_posts.map { |post| blog_item(post) }
     end
 
+    def blogs_filter_options
+      {
+        filter_name: 'theme',
+        options: filter_options
+      }
+    end
+
     protected
 
+    def filter_options
+      [
+        { label: t('global.actions.filter-all'), value: 'all' },
+        { label: Topic.find_by_slug('fashion').label, value: 'fashion' }
+      ].tap do |options|
```

Comment:
any way we can make use of the theme_filters method from the controller to populate this?